numba/cuda/tests/cudapy/test_caching.py

```diff
@@ -190,6 +203,15 @@ def test_non_writable_pycache(self):
 
         self._test_pycache_fallback()
 
+    def test_cannot_cache_linking_libraries(self):
+        link = os.path.join(Path(__file__).parent.parent,
+                            'cudadrv', 'data', 'jitlink.ptx')
```

Comment:
FYI, could have used `/` (slash operator) instead of join; e.g. `Path(__file__).parent.parent / 'cudadrv' / 'data' / 'jitlink.ptx'`

Reply:
Huh, interesting - I didn't know that!